lib/coarsecoarse_op_mma.in.cu

```diff
@@ -40,10 +40,10 @@ namespace quda {
       } else {
         GaugeFieldParam param(X);
         param.order = order;
-        output = cudaGaugeField::Create(param);
+        output = new GaugeField(param);
         if (copy_content) output->copy(X);
       }
-      return static_cast<cudaGaugeField *>(output);
+      return static_cast<GaugeField *>(output);
```

Comment:
Since now there is only `GaugeField`, we no longer need the `static_cast` anymore?

Reply:
Indeed (dd66595e1a5e2f61a04d5ec4dbc494b3da5840b4).